README.md

```diff
@@ -188,24 +189,27 @@ $ make CFG_TEE_CORE_LOG_LEVEL=4
 
 ---
 ### 4.4 STMicroelectronics boards
-Currently OP-TEE is supported on Orly-2 (`b2020-h416`) and Cannes family
-(`b2120` both `h310` and `h410` chip).
+Currently OP-TEE is supported on Orly-2 (`b2020-h416`), Cannes family
+(`b2120` both `h310` and `h410` chip) and 96boards/cannes board (`b2260-h410`).
 
-#### 4.4.1 Get the compiler for Orly-2
-Will be written soon.
+#### 4.4.1 Get the compiler
+Follow the instructions in the "4.2 Basic setup".
 
 #### 4.4.2 Download the source code
 See section "4.2.2 Download the source code".
 
-#### 4.4.3 Build for Orly-2
-For Orly-2 do as follows
+#### 4.4.3 Build the images and files
+For the 96boards/cannes:
 ```
-$ PLATFORM=stm-orly2 CROSS_COMPILE=arm-linux-gnueabihf- make
+$ make PLATFORM=stm-b2260 CROSS_COMPILE=arm-linux-gnueabihf-
```

Comment:
`CROSS_COMPILE=arm-linux-gnueabihf-` is supposed to be the default, can you check and remove it? Same below (twice).